src/Modules/Macros.asm

```diff
@@ -253,3 +253,9 @@ Divide_By_4 MACRO ih,il,oh,ol
     rrc  A
     mov  ol, A
 ENDM
+
+Divide_By_2_Inplace MACRO hi, lo
```

Comment:
If possible it would be preferable to use the same wording like: Divide_11Bit_By_8, in this case Divide_16Bit_By_2

Reply:
Makes sense - fixed.